pages/_app.tsx

```diff
@@ -34,9 +34,6 @@ function MyApp(props: MyAppProps) {
 
           <link rel='apple-touch-icon' href='favicon_152x152.png'/>
           <meta name='msapplication-square150x150logo' content='favicon_152x152.png'/>
-
-            <meta property="og:image" content="https://expenses-dev.makerdao.network/favicon.png"/>
-            <meta name="twitter:image" content="https://expenses-dev.makerdao.network/favicon.png"/>
```

Comment:
I understand this is necessary for the core unit about image to be a custom one, but. Could you move it to core unit tables instead of removing it?

Reply:
yes , of course, i will move